app/models/mapping.rb

```diff
@@ -28,6 +28,10 @@ def self.lookup(site, id)
     find_by(external_site: site, external_id: id).try(:item)
   end
 
+  def self.guess_algolia(type, query)
```

Comment:
Algolia is just how things work from now on, so there's no need to have an `_algolia` suffix.  Probably the best solution here is to just do something like:

```ruby
def self.guess(type, query)
  query = query[:title] if query.is_a?(Hash) # Backwards compat
  AlgoliaMediaIndex.search(query, type).first
end
```

So the old calls to `.guess()` keep working until we strip it down (or alternatively we can bring back the count matching if Algolia doesn't match perfectly)